var/spack/repos/builtin/packages/sundials/package.py

```diff
@@ -39,6 +39,10 @@ class Sundials(CMakePackage):
     # Versions
     # ==========================================================================
 
+    version('develop', '1d4b538721b84ebc91ce7ad92d94beae',
+            url='https://computation.llnl.gov/projects/sundials/download/sundials-4.0.0-dev.tar.gz')
```

Comment:
The current 4.0.0-dev release will be replaced with subsequent development releases as updates are made for the final 4.0.0 release later this this year. When a new snapshot is posted, the current development tarball will be removed from the download site. In this case I figured it was better to have the version as ```@develop``` rather than a specific version number that will be removed in the future.

Reply:
But then the checksum for version `develop` will change, right? Wouldn't it be better to remove the version altogether once 4.0.0 is released? We add alpha/beta/release candidate versions of other packages and later remove them as well.

Reply:
I hadn't thought about the checksum, that's a good point. I'll remove ```develop``` and add a ```4.0.0-dev``` version with the ```3.1.1 ``` set as the preferred version.